packages/orbit-components/src/Separator/index.tsx

```diff
@@ -1,77 +1,79 @@
 "use client";
 
 import * as React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import type * as Common from "../common/types";
-import type { Theme } from "../defaultTheme";
-import defaultTheme from "../defaultTheme";
-import getSpacingToken from "../common/getSpacingToken";
-import { left, right } from "../utils/rtl";
-import type { Props, Indent, Align } from "./types";
-import useTheme from "../hooks/useTheme";
+import type { Props, Align } from "./types";
+import { getSpaceAfterClasses } from "../common/tailwind";
 
-function capitalize(string: string) {
-  return string.charAt(0).toUpperCase() + string.slice(1);
+enum BorderTypes {
+  none = "border-none",
+  solid = "border-solid",
+  dashed = "border-dashed",
+  dotted = "border-dotted",
+  double = "border-double",
 }
 
-function getIndentAmount({
-  theme,
-  $indent,
-}: // StyledContainer takes more props, so this shouldn't be exact
-{
-  theme: Theme;
-  $indent: Indent;
-}) {
-  return $indent === "none" ? 0 : theme.orbit[`space${capitalize($indent)}`];
+enum Indent {
+  none = "none",
+  small = "small",
+  medium = "medium",
+  large = "large",
+  XLarge = "XLarge",
+  XXLarge = "XXLarge",
```

Comment:
I'd say we don't need this enum 🤔 We can say the function's parameter is of the same type as the prop

Reply:
I did it in the same way as we have it in `common/tailwind` folder

There is one thing for the future, that's out of the scope of this PR, but this can be useful for future Orbit optimization with TW, but there are two things that need to be improved and keep in mind: 

- types like this (Indent above) should be `const enum`, not just `enum`
- we need to add `optimizeConstEnums: true` to our `@babel/preset-typescript`

so after build we will have output like this (inlined primitive values, but in the code we still can have enums, that will be removed after build)  
<img width="409" alt="Screenshot 2023-11-01 at 20 03 07" src="https://github.com/kiwicom/orbit/assets/14054752/2afc8fbb-531b-4a3c-937a-efd9c4980cf6">